optimum/bettertransformer/models/base.py

```diff
@@ -61,6 +82,13 @@ def __init__(self, config):
                 self.num_layers = getattr(config, attr)
                 break
 
+        if orig_layer is not None:
+            # Last step, store the old module skeleton by copying the old module and putting
+            # it on the `meta` device.
+            self.orig_layer = deepcopy(orig_layer).to("meta")
```

Comment:
The `deepcopy(orig_layer)` sounds like it would make a copy in RAM / GPU memory before being set on meta device. I would if `deepcopy(orig_layer.to("meta"))` is not better (if we absolutely need deepcopy)

Reply:
I tried and didn't worked, I guess it's because `.to` is an in-place operation